src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfterArrayDeclarations3.java

```diff
@@ -0,0 +1,11 @@
+package com.puppycrawl.tools.checkstyle.checks.whitespace.nowhitespaceafter;
+
+import org.eclipse.jdt.annotation.NonNull;
```

Comment:
Implement your own private interface into this input.
Example: https://github.com/checkstyle/checkstyle/blob/4d1d39f6db117665ec5210e9f885bd2c197f4313/src/test/resources/com/puppycrawl/tools/checkstyle/grammars/java8/InputAnnotations3.java#L15